visualizer/parseTree.js

```diff
@@ -717,8 +718,10 @@
   // -------
 
   var parseTree = ohmEditor.parseTree = new CheckedEmitter();
-  parseTree.refresh = refreshParseTree;
-
+  parseTree.refresh = function(rootTrace, clearZoomState) {
+    clearMarks();
```

Comment:
Sounds good to me. I try to move it, and the test is complaining about cannot find `cmUtil.clearMark`, not sure how you want to handle this, so I just keep it here for now.